.svnignore

```diff
@@ -7,8 +8,6 @@ package.json
 phpunit.xml
 PUBLISHING.md
 README.md
-vipgo-helper.php
-wpcom-helper.php
```

Comment:
Why remove these two?

Reply:
Yeah, I realized that mistake after and restored them in a future PR.

Reply:
Cool, going to merge

Reply:
Oh, sorry, I totally misunderstood the question. We want to include these files for any VIPs that download the plugin from the .org repo (rather than GitHub).